assembl/static2/workspaces/draft-js-attachment-plugin/src/components/AddAttachmentForm.jsx

```diff
@@ -22,12 +22,13 @@ const AddAttachmentForm = ({ onSubmit }: Props) => (
   <Form
     onSubmit={onSubmit}
     render={({ handleSubmit }) => (
-      <BootstrapForm componentClass="div" horizontal>
-        <Field name="file" component={FileUploaderFieldAdapter} />
-        <FormGroup>
-          <Col smOffset={2} sm={10}>
+      <BootstrapForm>
```

Comment:
if you remove the `componentClass="div"`, it breaks the behavior. Because it creates a form in a form and react-final-form doesn't like that. Is it necessary?

Reply:
No it's not. My bad. I add it again.